### PR TITLE
[Site] New Icon component + assets optimisation

### PR DESCRIPTION
**Pull request by smnandre**
* new Icon twig component to handle SVG icons (and remove all FontAwesome scripts + fonts)
* clean & compress favicons
* expand header full-width on big screens
* clean footer
* fix layout + add UI effects on component list 
* fix some minor misalignments

### CODE REVIEW

File: ux.symfony.com/public/site.webmanifest [hunk 1]
```diff
@@ -1 +1,2 @@
-{"name":"","short_name":"","icons":[{"src":"/android-chrome-192x192.png","sizes":"192x192","type":"image/png"},{"src":"/android-chrome-512x512.png","sizes":"512x512","type":"image/png"}],"theme_color":"#ffffff","background_color":"#ffffff","display":"standalone"}
\ No newline at end of file
+
+{"name":"Symfony UX","short_name":"Symfony UX","icons":[{"src":"/android-chrome-192x192.png","sizes":"192x192","type":"image/png"},{"src":"/android-chrome-512x512.png","sizes":"512x512","type":"image/png"}],"theme_color":"#ffffff","background_color":"#ffffff","display":"standalone"}
```
Comment (weaverryan) [inline ux.symfony.com/public/site.webmanifest:2]: Was the empty line by accident?

File: ux.symfony.com/assets/styles/_variables.scss [hunk 2]
```diff
@@ -11,4 +11,24 @@ $n-800: #262626;
...
 $red: #FB4040;
 $yellow: #FBC640;
-$green: #84DE2C;
\ No newline at end of file
+$green: #84DE2C;
+
+
+
+
+@at-root :root {
```
Comment (weaverryan) [inline ux.symfony.com/assets/styles/_variables.scss:19]: There's some extra line breaks above this

File: ux.symfony.com/templates/main/_componentsList.html.twig [hunk 3]
```diff
@@ -1,22 +1,5 @@
...
-                        </div>
-                    </div>
-                </div>
-            </a>
-        </div>
-    </div>
-{% endfor %}
+<div class="" style="display: grid; gap: 2rem;  grid-template-columns: repeat(auto-fit, minmax(min(100%, 400px), 1fr));">
+    {% for package in packages %}
+        <twig:ComponentBox :package="package" />
```
Comment (weaverryan) [inline ux.symfony.com/templates/main/_componentsList.html.twig:3]: Well that's gorgeous

**Review by weaverryan — APPROVED** [review]
This is wonderful. I have 2 whitespace comments -but they're so minor, I'm going to merge now and deploy. Thank you!

**Comment by weaverryan** [comment]
Thanks @smnandre!

**Comment by smnandre** [comment]
> This is wonderful. I have 2 whitespace comments -but they're so minor, I'm going to merge now and deploy. Thank you!

I'll correct both in following PR, sorry for that :)

**Pull request merged**